terraform/environments/digital-prison-reporting/glue-connections.tf

```diff
@@ -0,0 +1,66 @@
+locals {
+  operational_db_jdbc_connection_string = "jdbc:postgresql://${module.aurora_operational_db.rds_cluster_endpoints["static"]}:${local.operational_db_port}/${local.operational_db_default_database}"
+  nomis_jdbc_connection_string          = "jdbc:oracle:thin:@${local.nomis_host}:${local.nomis_port}/${local.nomis_service_name}"
+}
+
+# Operational DataStore
+resource "aws_glue_connection" "glue_operational_datastore_connection" {
```

Comment:
Could you make these resources conditional, with a COUNT - able to manage per env Please